README.md

```diff
@@ -182,4 +182,16 @@ object to find references to the bpfMap pinpoints (`spec.maps`) to configure the
 ## Developer
 
 For more architecture details about `bpfman-operator`, refer to
-[Developing the bpfman-operator](https://bpfman.io/v0.5.0-rc1/developer-guide/develop-operator)
\ No newline at end of file
+[Developing the bpfman-operator](https://bpfman.io/v0.5.0-rc1/developer-guide/develop-operator)
+
+### Deploy as a bundle from the Console's OperatorHub page
+
+This mode is recommended when you want to test the customer experience of navigating through the operators'
+catalog and installing/configuring it manually through the UI.
+
+```sh
+export BUNDLE_IMG=quay.io/$USER/bpfman-operator-bundle:v$VERSION
```

Comment:
NIT, $VERSION may not exist here, either just recommend a dev tag or tell the user to set Version above

```suggestion
export BUNDLE_IMG=quay.io/$USER/bpfman-operator-bundle:development
```

Reply:
sure